variables.tf

```
@@ -55,6 +55,12 @@ variable "clear_text_terraform_variables" {
   description = "An optional map with clear text Terraform variables"
 }
 
+variable "create_backend_file" {
```

```suggestion
variable "create_backend_tf" {
```

Or maybe `create_backend_config`?

---

`create_backend_config` it is!